src/Blazored.Modal/wwwroot/blazored-modal.css

```diff
@@ -79,3 +79,22 @@
     bottom: 32px;
     right: 32px;
 }
+
+.blazored-modal-scrollable {
+    display: flex;
+    max-height: calc(100% - 1rem);
+}
+
+@media (min-width: 576px) {

```

Comment:
Do we need to add this media query? We've avoided adding these before as it's difficult to get right. Application requirements when they go responsive are hard to predict and often quite bespoke.

Reply:
It can come out, I just included it because bootstrap did it. 
It just puts more space at the top of the screen on larger screens.